### PR TITLE
[PAE 117] Add utm parameters to salesforce

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Get utm parameters from the url or cookie and attach them to the salesforce api call.
jira ticket https://pearsonadvance.atlassian.net/browse/PAE-117

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -10,6 +10,7 @@ django==1.11.26
 django-compressor==2.2
 django-cors-headers                 #  Used to allow to configure CORS headers for cross-domain requests
 django-crispy-forms
+django-crum
```
Comment (diegomillan) [inline requirements/base.in:13]: Would it be useful to pin django-crum version?? @andrey-canon

**Review by diegomillan — COMMENTED** [review]
Please review a minor comment @andrey-canon

File: ecommerce/extensions/analytics/utils.py [hunk 2]
```diff
@@ -204,3 +205,29 @@ def get_google_analytics_client_id(request):
         return '.'.join(google_analytics_cookie.split('.')[2:])
 
     return None
+
+
+def get_utm_session_parameters():
+    """
+    Return utm parameters from the cookie or current url request.
+    The url values have priority.
```
Comment (Squirrel18) [inline ecommerce/extensions/analytics/utils.py]: This is not applying. @andrey-canon

Comment (andrey-canon) [reply ecommerce/extensions/analytics/utils.py]: ??

File: requirements/base.in (hunk 1 above)
Comment (andrey-canon) [reply requirements/base.in:13]: the version is specified in the txt file, the .in file shouldn't contain pinned versions

File: ecommerce/extensions/analytics/utils.py (hunk 2 above)
Comment (Squirrel18) [reply ecommerce/extensions/analytics/utils.py]: You say the URL params have priority but the code is doing nothing to do what it is saying here. @andrey-canon This comment is not applying, please remove it.

Comment (andrey-canon) [reply ecommerce/extensions/analytics/utils.py]: line 227
request.COOKIES is a dict for example {'utm': 'pepito', ........} after line 226 data = {'utm': 'pepito', ........}, if the url is 
http://daniel.com/?utm=little-daniel, request.GET.dict() = {'utm': 'little-daniel'}, after the line 227
 data will be  {'utm': 'little-daniel'}

Comment (Squirrel18) [reply ecommerce/extensions/analytics/utils.py]: By overwritten, but I think is better to explain explicitly, URL parameters will overwrite Cookie parameters. @andrey-canon

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**